spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Optimizer.scala

```diff
@@ -284,6 +286,33 @@ abstract class Optimizer[T: ClassTag, D](
   }
 
   def prepareInput(): Unit = {}
+
+  /**
+   * Set constant gradient clipping
+   * @param min the minimum value to clip by
+   * @param max the maximum value to clip by
+   * @return
+   */
+  def setConstantGradientClipping(min: Float, max: Float)
```

Comment:
any way to disable clipping? People might use clipping in some stages of training and then diable it later. Also optimizer could be reused. Need a way to set to defaults.

Reply:
Added disable clipping method.